flink-runtime/src/test/java/org/apache/flink/runtime/zookeeper/ZooKeeperStateHandleStoreTest.java

```diff
@@ -693,66 +770,79 @@ protected void setStateHandle(
         store.addAndLock(
                 pathInZooKeeper, new TestingLongStateHandleHelper.LongStateHandle(initialState));
 
-        try {
-            store.replace(
-                    pathInZooKeeper,
-                    IntegerResourceVersion.valueOf(0),
-                    new TestingLongStateHandleHelper.LongStateHandle(replaceState));
-            fail("Did not throw expected exception");
-        } catch (Throwable t) {
-            assertThat(t, IsInstanceOf.instanceOf(expectedException));
-        }
+        assertThatThrownBy(
+                        () ->
+                                store.replace(
+                                        pathInZooKeeper,
+                                        IntegerResourceVersion.valueOf(0),
+                                        new TestingLongStateHandleHelper.LongStateHandle(
+                                                replaceState)))
+                .isInstanceOf(expectedException);
```

Comment:
```suggestion
                .isInstanceOf(expectedExceptionClass);
```
can we rename the parameter as well?